tools/fix-ansible.py

```diff
@@ -0,0 +1,10 @@
+import os
+import sys
+
+for handle in (sys.stdin, sys.stdout, sys.stderr):
+    try:
+        fd = handle.fileno()
+    except Exception:
+        continue
+
+    os.set_blocking(fd, True)
```

Comment:
I would rename the script to `ansible-blocking-io` or something similar and add a comment pointing to the ansible issue that's linked in the PR description (`#80108`).  From what I understand, this isn't a bug in anisble itself but a side-effect of a different change (either in ansible or somewhere else), so it'll be good to have some information about why we're doing this for the future.

Reply:
Done, renamed to ansible-blocking-io.py, and added more info in it about why I am doing this.